unit/docker.env

```diff
@@ -0,0 +1,5 @@
+#LS_JAVA_OPTS="-Xms256m -Xmx256m -XX:MaxMetaspaceSize=256m"
+# Common LS specific testing options (can be overriden from ENV e.g. in .travis.yml) :
+# - `-X-C` disabled JIT compilation for unit tests (-Xcompile.mode=OFF)
+# - `-v -W1` print JRuby version but do not go verbose
+JRUBY_OPTS=-Xregexp.interruptible=true -Xcompile.invokedynamic=false -X-C -J-XX:+PrintCommandLineFlags -v -W1
```

Comment:
can we just do `--dev` here? (+ -v -W1)

Reply:
`--dev` is a bit more aggressive (did not want to turn off bytecode verification).
thought it might make sense to be explicit and leave most of the other stuff as is.
was also contemplating running (even unit tests) with `-Xcompile.invokedynamic=true`

but I am biased here (knowing indy is a bit under tested at JRuby's end :neutral_face:), 
maybe all confusing & simply having "faster unit test runs with --dev is all we really care for" ?

Reply:
> faster unit test runs with --dev is all we really care for"

Phrasing it this way makes me think.. one of my concerns is that we can have bugs not be triggered because we're optimizing for startup time. 

Turning off compilation+indy is helpful in places like the plugin manager, that are one shot commands. Our testing suites take 10 minutes or more to run, so I wonder if it's worth at all using turning off compilation here.